### DISCUSSION TITLE
How to set window border color?

### DISCUSSION THREAD

**Discussion by billywatkins** [tauri-apps/tauri#8517] [Q&A] [answered]
Hello,

I am trying to set the window color, but I can't find any API to do that. Can this be achieved with https://crates.io/crates/cocoa?

I want something similar to how Spacedriveapp has a border color, which is also written in Tauri.

![image](https://github.com/tauri-apps/tauri/assets/124872003/9c6199c1-bb26-4dc4-8c07-a995eac978cb)

**Comment by FabianLars** [comment, marked as answer]
The window _border_ color looks like the default to me (if you disabled decorations you may need https://github.com/tauri-apps/window-shadows to get the borders back). The window _background_ color to me looks like it's done via html/css (having `transparent: true` in the tauri window config)

**Reply by billywatkins** [reply]
thanks!
